test/test-pages/ehow-2/expected.html

```diff
@@ -0,0 +1,227 @@
+                </a>
+            </li>
+        </ol>
+
+        <header class="page-head bordered pre-col">
+
+            <div data-type="AuthorProfile">
+                <div class="post-meta clearfix headline6 mg-2">
+                    <a class="gtm_contributorBylineAvatar contributor-follow-tip" id="img-follow-tip" href="http://fakehost/contributor/gina_robertsgrey/" target="_top">
+                        <img src="http://img-aws.ehowcdn.com/60x60/cme/cme_public_images/www_demandstudios_com/sitelife.studiod.com/ver1.0/Content/images/store/9/2/d9dd6f61-b183-4893-927f-5b540e45be91.Small.jpg" class="avatar fl" data-failover="//img-aws.ehowcdn.com/60x60/ehow-cdn-assets/test15/media/images/authors/missing-author-image.png" onerror="var failover = this.getAttribute('data-failover');
+                if (failover) failover = failover.replace(/^https?:/,'');
+                var src = this.src ? this.src.replace(/^https?:/,'') : '';
+                if (src != failover){
+                    this.src = failover;
+                }" /> </a>
+                </div>
+                <div class="post-meta clearfix headline6 mg-2" id="author_powertip" data-author-url="/contributor/gina_robertsgrey/">
+                    <a class="gtm_contributorBylineAvatar" href="http://fakehost/contributor/gina_robertsgrey/" target="_top">
+                        <img src="http://img-aws.ehowcdn.com/60x60/cme/cme_public_images/www_demandstudios_com/sitelife.studiod.com/ver1.0/Content/images/store/9/2/d9dd6f61-b183-4893-927f-5b540e45be91.Small.jpg" class="avatar fl" data-failover="//img-aws.ehowcdn.com/60x60/ehow-cdn-assets/test15/media/images/authors/missing-author-image.png" onerror="var failover = this.getAttribute('data-failover');
+                if (failover) failover = failover.replace(/^https?:/,'');
+                var src = this.src ? this.src.replace(/^https?:/,'') : '';
+                if (src != failover){
+                    this.src = failover;
+                }" /> </a>
+                    <p class="btn btnFollow" data-type="contributor" data-author-url="/contributor/gina_robertsgrey/" data-follow-data="{&quot;name&quot;:&quot;Gina Roberts-Grey&quot;,&quot;slug&quot;:&quot;\/contributor\/gina_robertsgrey\/&quot;,&quot;image_url&quot;:&quot;http:\/\/s3.amazonaws.com\/cme_public_images\/www_demandstudios_com\/sitelife.studiod.com\/ver1.0\/Content\/images\/store\/9\/2\/d9dd6f61-b183-4893-927f-5b540e45be91.Small.jpg&quot;,&quot;website&quot;:&quot;&quot;}">Follow</p>
+                </div>
+                <p class="article-meta">
+                    <time datetime="2016-09-14T07:07:00-04:00" itemprop="dateModified">Last updated September 14, 2016</time>
+                </p>
+            </div>
+            <div class="clearfix mg-3">
+                <ul class="social-icons js-social-icons inline " data-url="http://www.ehow.com/how_4851888_throw-graduation-party-budget.html" share-buttons-uninitialized="">
```

Comment:
Sure, I think we can add `social` into the `unlikelyCandidates` REGEXP to remove it.